packages/react-scripts/template/src/index.js

```diff
@@ -1,13 +1,21 @@
+/* eslint-disable react/jsx-filename-extension */
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Router } from '@reach/router';
 import './index.css';
 import App from './components/App';
 import * as serviceWorker from './serviceWorker';
 
-// eslint-disable-next-line react/jsx-filename-extension
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render(
+  <Router>
+    <App path="/" />
+  </Router>,
```

Comment:
is that trailing comma important for anything?

Reply:
Yes, it separates the function parameters :)